### PR TITLE
Prod mode

### PR DESCRIPTION
**Pull request by jorge07**
Allow app to run in prod mode by running `make env=prod start`

### CODE REVIEW

**Comment by jorge07** [comment]
Need squash

File: etc/prod/docker-compose.windows.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+version: '3.7'
+
+services:
+
+  php:
+    environment:
+      XDEBUG_CONFIG: remote_connect_back=0 remote_host=host.docker.internal remote_port=9000
```
Comment (cv65kr) [inline etc/prod/docker-compose.windows.yml:7]: Xdebug in production?

File: etc/prod/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+  nginx:
+    ports:
+      - "80:80"
+
+  php:
+    ports:
+      - "2323:22"
+    # Allows to debug php script run inside PHP container
+    environment:
```
Comment (cv65kr) [inline etc/prod/docker-compose.yml:13]: maybe `env_file`?

File: etc/prod/docker-compose.windows.yml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+version: '3.7'
```
Comment (cv65kr) [inline etc/prod/docker-compose.windows.yml:1]: Maybe this a great opportunity for bump docker-compose version to `3.8`
https://docs.docker.com/compose/compose-file/compose-versioning/#version-38

File: etc/prod/docker-compose.windows.yml (hunk 1 above)
Comment (Lutacon) [reply etc/prod/docker-compose.windows.yml:7]: It says windows in name file.

Comment (jorge07) [reply etc/prod/docker-compose.windows.yml:7]: Why not xdebug? This is about run the application in production mode, not a deploy process.

File: etc/prod/docker-compose.yml (hunk 2 above)
Comment (jorge07) [reply etc/prod/docker-compose.yml:13]: May work. Will have a look

File: etc/prod/docker-compose.windows.yml (hunk 3 above)
Comment (jorge07) [reply etc/prod/docker-compose.windows.yml:1]: Will require people to upgrade locally and ci for no benefits right now

**Comment by jorge07** [comment]
I've tested env_file and didn't like it too much. Will iterate in another PR

**Pull request merged**